bin/tx-prover/README.md

```diff
@@ -53,6 +53,13 @@ PROVER_SERVICE_PORT=<your-port>
 RUST_LOG=<log-level>
 ```
 
+## Features
+
+For the service, the following features are available:
+
+- `testing`: This flag reduces the difficulty target of the account seed creation. If the client uses the `testing` flag, then the prover will not be able to prove transactions correctly without it.
+- `concurrent`: Enables transaction proving speedups.
```

Comment:
Should we list all features here (similar to how we do it [here](https://github.com/0xPolygonMiden/miden-base/tree/main/objects#features))?

Also, for concurrent, I'd say something like: "implies `std` and also enables multi-threaded proof generation."

Reply:
Done